CorsixTH/Lua/app.lua

```diff
@@ -1470,7 +1470,7 @@ function App:getVersion(version)
   if ver > 156 then
     return "Trunk"
   elseif ver > 138 then
-    return "v0.65"
+    return "v0.65.1"
```

Comment:
might be best to leave it - we don't know whether they had the hotfix or not.

Reply:
```suggestion
    return "v0.65"
```